src/UsgsAstroLsSensorModel.cpp

```diff
@@ -2438,16 +2364,17 @@ void UsgsAstroLsSensorModel::getAdjSensorPosVel(
 //***************************************************************************
 // UsgsAstroLineScannerSensorModel::computeViewingPixel
 //***************************************************************************
-csm::ImageCoord UsgsAstroLsSensorModel::computeViewingPixel(
+std::vector<double> UsgsAstroLsSensorModel::computeDetectorView(
    const double& time,
    const csm::EcefCoord& groundPoint,
    const std::vector<double>& adj,
    const double& desiredPrecision) const

```

Comment:
desiredPrecision can be removed as an argument because this doesn't do anything iterative anymore.